github_server/action.yml

```diff
@@ -1,53 +1,143 @@
 name: "ALKiln/github_server: Isolated docassemble tests"
-description: "Automatically test docassemble interviews on GitHub's servers, in an isolated environment. Avoids docassemble reload errors and other external factors"
+description: "Download and start up a temporary docassemble server on GitHub's servers, in an isolated environment. Avoids docassemble reload errors and other external factors."
```

Comment:
Change the name of this folder to `action_github_server`

(As per main comment item 2)